app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java

```diff
@@ -304,6 +307,23 @@ private void importData(String path) {
         }
     }
 
+    /**
+     * Toggle menu items.
+     *
+     * @param item
+     * @param visibility
+     */
+    private void toggleMenuItem(MenuItem item, boolean visibility) {
+        item.setVisible(visibility);
```

Comment:
Come on. Not like this. Since only one icon is shown we can set visibility using ! operator.
```
menu.findItem(R.id.play_graph), visibility
menu.findItem(R.id.play_graph), !visibility
```

Reply:
@CloudyPadmal But we need to toggle visibility for `stop_data` as well at the same time, so that the play and stop buttons appear at the same place on the app bar simultaneously. I can create a boolean variable. Do let me know.

Reply:
Okay let me write the function for you. This is basic Java.

```java
private void toggleMenuItem(Menu menu, boolean isPlaying) {
    MenuItem play = menu.findItem(R.id.play_graph);
    play.setVisible(isPlaying);
    MenuItem stop = menu.findItem(R.id.stop_data);
    stop.setVisible(isPlaying);
    this.invalidateOptionsMenu();
}
```

Call this function in when playing `toggleMenuItem(menu, true);`

Reply:
Ah! Misunderstood stuff. I thought you were directing to only toggle `play_graph`. Thanks have updated.